activities/FoodChain.activity/lib/onyx/package.js

```diff
@@ -1,8 +1,7 @@
 enyo.depends(
 	// $lib/onxy loads the debug code for now, probably it should load the built code as below
-	"source"
-	/*
-	"build/onyx.css",
-	"build/onyx.js"
-	*/
+	"source",
+	"bootstrap-tour-standalone.min.css",
```

Comment:
You should not update this file, including CSS and JS files in the index.html is sufficient